pages/features.js

```diff
@@ -11,13 +15,10 @@ import Hero from '../src/Hero'
 import FeaturesHero from '../src/FeaturesHero'
 import TrySection from '../src/TrySection'
 
-const HeadInjector = () => (
-  <Head>
-    <title>Features | Machine Learning Version Control System</title>
-  </Head>
-)
-
-export default () => (
+/**
+ * FeaturesPage fn component
+ */
+const FeaturesPage = () => (
```

Comment:
Is there any reason I shouldn't turn these into an `export default function...` statement instead of an arrow function that gets exported later? To be consistent with components in src/

Reply:
For example I know `this` behaves differently in arrow functions, but not sure whether that or something else is a reason for this. I'd prefer to have consistency if not (switch to `export default function` which seems to be more widely used in our React code base).

cc @iAdramelk @shcheklein thanks for feedback is possible, still learning React over here.

Reply:
We should change them all to the same styles, yes. There is no practical reason there to use arrow function instead of normal function.

Reply:
Changing according to https://github.com/airbnb/javascript/tree/master/react#naming and [...#parentheses](https://github.com/airbnb/javascript/tree/master/react#parentheses)...